.flake8

```
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
```

```suggestion
ignore = E203, E266, E501, W503, F403, F401, E711
```

E711 is a little dangerous right now because of this problem with the bindings: https://github.com/apple/foundationdb/issues/330

---

Yes, great that you noticed that!